03-Bootstrap/assignment/index.html

```diff
@@ -1,15 +1,170 @@
+      integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO"
+      crossorigin="anonymous"
+    />
+
+    <!-- Custom CSS -->
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Assignment 03 - Bootstrap 4</title>
+  </head>
+  <body>
+    <main class="main">
+      <div class="container-fluid">
+        <header class="d-flex flex-wrap justify-content-center py-3">
+          <h1 class="display-4 text-center">Assignment 03 - Bootstrap 4</h1>
+        </header>
+
+        <nav
+          class="navbar navbar-expand-lg navbar-light bg-info d-flex justify-content-center py-3"
+        >
+          <ul class="navbar-nav">
+            <li class="nav-item active">
+              <a class="nav-link" href="index.html">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="images.html">Images</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="accordion.html">Accordion</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="grid.html">Grid</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+
+      <div class="container hero">
+        <section class="section">
+          <h2 class="display-3 text-center">Home</h2>
+          <p class="display-4 text-center">
+            Welcome to the home page of Assignment 03 - Bootstrap 4.
+          </p>
+          <div class="row">
+            <div class="col-md-6">
+              <p class="text-danger text-center">
+                This is a simple website that demonstrates the use of Bootstrap
+                4.
+              </p>
+              <div class="d-flex justify-content-center">
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="280"
+                  height="280"
+                  fill="currentColor"
+                  class="bi bi-bootstrap"
+                  viewBox="0 0 16 16"
+                >
+                  <path
+                    d="M5.062 12h3.475c1.804 0 2.888-.908 2.888-2.396 0-1.102-.761-1.916-1.904-2.034v-.1c.832-.14 1.482-.93 1.482-1.816 0-1.3-.955-2.11-2.542-2.11H5.062zm1.313-4.875V4.658h1.78c.973 0 1.542.457 1.542 1.237 0 .802-.604 1.23-1.764 1.23zm0 3.762V8.162h1.822c1.236 0 1.887.463 1.887 1.348 0 .896-.627 1.377-1.811 1.377z"
+                  />
+                  <path
+                    d="M0 4a4 4 0 0 1 4-4h8a4 4 0 0 1 4 4v8a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4zm4-3a3 3 0 0 0-3 3v8a3 3 0 0 0 3 3h8a3 3 0 0 0 3-3V4a3 3 0 0 0-3-3z"
+                  />
+                </svg>
+              </div>
+            </div>
+            <div class="col-md-6">
+              <p class="text-danger text-center">
+                This website includes a home page, an images page, an accordion
+                page, and a grid page.
+              </p>
+              <div class="col" style="border: 1px solid red">
```

Comment:
Try to not use insline styles like this. Instead either use a bootstrap class if available or find a way to do this with css